src/desktop/components/search_bar/view.coffee

```diff
@@ -47,9 +48,11 @@ module.exports = class SearchBarView extends Backbone.View
     @search = new Search restrictType: @restrictType, mode: @mode, fairId: @fairId, size: @limit, agg: @enableAggregations
 
     @on 'search:start', @indicateLoading
+    @on 'search:start', @recordPerformanceMeasurementStart
```

Comment:
#minor I think you can do `@on 'search:start', '@recordPerformanceMeasurementStart if @shouldMeasurePerformance` and similarly in the other place (and skip the checks below).